lib/.eslintrc.yaml

```diff
@@ -163,6 +163,7 @@ rules:
   node-core/lowercase-name-for-primitive: error
   node-core/non-ascii-character: error
   node-core/no-array-destructuring: error
+  node-core/avoid-prototype-pollution: error
```

Comment:
Non-blocking nit: The other rules are all in alphabetical order, so maybe consider moving this and/or renaming it? (`no-pollutable-prototypes` or something like that?)